src/util/token.js

```diff
@@ -3,17 +3,9 @@ const crypto = require('crypto') // sodium doesn't have HMAC
 const base64url = require('base64url')
 
 const TOKEN_HMAC_INPUT = 'token'
+const AUTH_HMAC_INPUT = 'authorization'
```

Comment:
all we're doing with this is key-stretching, right? The bearer token derived from the shared secret will be reused over and over (so eavesdrop the https connection once, and you can forever create discord between the two users of this trustline), but by hashing with this string, at least the attacker will not obtain the shared secret itself.

can you describe what the difference is between what an attacker can do with the auth token, and what they can do with the shared secret? What resource are we protecting here?